CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+# 1.0.8.4 (2019-11-1)
+## Edge Hub
+### Bug Fixes
+* Fix Twin property 4K length validation ([c4a97a4](https://github.com/Azure/iotedge/commit/c4a97a40e0b1db1ceb1483baa7dab6b17705467b))
+* Updating Twin update limits to match IoTHub ([0ae34d3](https://github.com/Azure/iotedge/commit/0ae34d3a8b6750e9caae8045b71cd21d2e40e119))
```

Comment:
Are we supposed to take the beginning or end segment of the commit?

Reply:
I checked from 1.0.8.2 where Mike and I did it and should be the beginning. I'll update.

Reply:
Actually, I'm not a contributor to your repo so I won't have access rights. Can you fix?